### DISCUSSION TITLE
Add direct link for Changelog and Roadmap

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#11413] [Mobile]
If there's a Changelog (e.g. CHANGELOG.md) or a roadmap (e.g. ROADMAP.md) present, show a direct link in the menu.

For the desktop version it would be handy too.

**Comment by brianlovin** [comment]
Thanks for the suggestion! I was aware of the common `CHANGELOG.md` style, but hadn't seen too many repos use the `ROADMAP.md` system — do you have an example of a repo using this?

**Reply by Zerotask** [reply]
You can search on Google:
ROADMAP.md site:github.com

It's mostly smaller repos. Changelog shows the past and Roadmap the future :D
Therefore those elements could be placed around the current releases for example

Besides that, CONTRIBUTING / CONTRIBUTE and UPGRADE md files are also very important.
Contributing is usually the first file you read before your do changes for FOSS and UPGRADE is especially useful for businesses using FOSS

**Reply by brianlovin** [reply]
Makes sense! We also dug this up, which has a handful of other recommended/special md files: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file
